Makefile

```diff
@@ -7,8 +7,10 @@
 # Note: if these files have been downloaded through the releases directory then
 # they need to be renamed with the appropriate {u-boot,linux}-xlnx- prefix so
 # that the file name and contents match.
-MD5_SUM_u-boot-xlnx-xilinx-v2015.1 = b6d212208b7694f748727883eebaa74e
-MD5_SUM_linux-xlnx-xilinx-v2015.1  = 930d126df2113221e63c4ec4ce356f2c
+MD5_SUM_u-boot-xlnx-xilinx-v2015.1 	= b6d212208b7694f748727883eebaa74e
```

Comment:
Don't change the whitespace on the existing MD5_SUM lines, so they don't appear in the diff